apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -68,10 +67,13 @@ const SetUpMessage = ({
   isRtl,
   headingText,
   descriptionText,
+  className,
   buttonText,
   buttonUrl,
+  buttonClass,
+  onClick,
 }) => (
-  <div style={styles.section}>
+  <div style={styles.section} className={className}>
```

Comment:
It looks like this is just used by tests? Not sure how strongly I feel about this, but adding props just for tests seems unfortunate. I wonder if we can/should instead formulate a more complicated selector (either give a parent a className and then find this element within it, or give this a fixed className, and do something like select the nth instance).